### PR TITLE
gcp push button documentation

### PR DESCRIPTION
**Pull request by karmab**

### CODE REVIEW

**Comment by scollier** [comment]
@karmab I reviewed this. It looks good. I'll leave it here until the image is published and I can actually walk through the instructions.

File: pages/cloud.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+layout: page
+title: Easy Install Using A Public Cloud Provider
+---
+
+You can try Kubevirt in one of the public clouds that are supported. The
```
Comment (scollier) [inline pages/cloud.md]: We should probably add a note at the top here about this being a "proof of concept" / "trial" instance.  Not meant for production.

File: pages/cloud.md [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+**Source:** cloud storage file
+
+**Cloud storage file:** kubevirt/{{release.tag}}.tar.gz
+
+### Step 2: Create a new instance using the image you created
+
+Once the image is created, you can create a new instance based on this image.
+Go to "Compute Engine", "VM instances and then click on "Create instance".
+
+![screenshot0042](/assets/images/kubevirt-button/create_instance_1.png.png){:.large_img}
```
Comment (scollier) [inline pages/cloud.md]: Not sure if it's the way github is rendering the "preview" of the page, but these screenshots are not showing properly.

File: pages/cloud.md [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+
+Name it *kubevirt*
+It's recommended to select the 2 CPU / 7.5GB instance. Under "boot disk",
+select the image that you created above
+
+![screenshot0042](/assets/images/kubevirt-button/create_instance_2.png.png){:.large_img}
```
Comment (scollier) [inline pages/cloud.md]: Not sure if it's the way github is rendering the "preview" of the page, but these screenshots are not showing properly.

File: pages/cloud.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+**Source:** cloud storage file
+
+**Cloud storage file:** kubevirt/{{release.tag}}.tar.gz
+
+### Step 2: Create a new instance using the image you created
+
+Once the image is created, you can create a new instance based on this image.
+Go to "Compute Engine", "VM instances and then click on "Create instance".
+
+![screenshot0042](/assets/images/kubevirt-button/create_instance_1.png){:.large_img}
```
Comment (scollier) [inline pages/cloud.md]: Now the images show, but the:

```
{:.large_img}
``` 
Is also showing, for both images.

File: pages/cloud.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+
+Name it *kubevirt*
+It's recommended to select the 2 CPU / 7.5GB instance. Under "boot disk",
+select the image that you created above
+
+![screenshot0042](/assets/images/kubevirt-button/create_instance_2.png){:.large_img}
```
Comment (scollier) [inline pages/cloud.md]: Now the images show, but the:

{:.large_img}
Is also showing, for both images.

File: pages/cloud.md [hunk 6]
```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: Easy Install Using A Public Cloud Provider
+---
+
+You can try Kubevirt in one of the public clouds that are supported. The
```
Comment (rwsu) [inline pages/cloud.md]: s/Kubevirt/KubeVirt

File: pages/cloud.md [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+benefits of this option are that you don't need any hardware yourself, and that
+you can also use the same public cloud as the platform to be managed.
+Note this setup is not meant for production, only to give you a quick taste of kubevirt's functionality
```
Comment (rwsu) [inline pages/cloud.md]: s/Kubevirt/KubeVirt

File: pages/cloud.md [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+In the instructions below we will use the Google Cloud Platform. The Kubevirt
```
Comment (rwsu) [inline pages/cloud.md]: s/Kubevirt/KubeVirt

**Comment by karmab** [comment]
@rwsu fixed the typos. need to look if there s a way to have the link directly fill some fields, yes

**Comment by rwsu** [comment]
@karmab nice! Can we also change the filename from cloud.md to gcp.md since there will be a separate page for AWS?

**Comment by karmab** [comment]
changed the name to gcp.md and put the correct link in layout page

**Pull request merged**